src/sxastarter/package.json

```diff
@@ -86,6 +86,8 @@
     "scaffold": "ts-node --project tsconfig.scripts.json scripts/scaffold-component.ts",
     "start:connected": "npm-run-all --serial bootstrap --parallel next:dev start:watch-components",
     "start:production": "npm-run-all --serial bootstrap next:build next:start",
+    "build:production": "npm-run-all --serial bootstrap next:build",
```

Comment:
What is the difference between "run" and "start" commands for XMC?
I don't have full context on these changes, but wanted to provide some info.

1. We already have a `build` command (above) that is the same as this `build:production`. This name is a common standard (for JSS and JS in general)
2. These scripts also accommodate local development and deployment on Vercel
3. Vercel has build and start phases and will use the [`build` script by default](https://vercel.com/docs/concepts/deployments/configure-a-build#build-command) and then `next start`

Reply:
@ambrauer We need to split the start:production command, without breaking how we run today, while allowing the developer to override the behavior.

Today, at runtime, we call start:production in Deploy. Today, at build time, we only call the lint command.

Instead, we want to build at build time, and run at runtime.

This is to reduce our memory footprint of the rendering host, and to fail sooner for the build in Deploy.

Reply:
@OlmStc I think the start:production should call build:production and then run:production

Reply:
@iamandycohen Got it, and that makes sense (and this is what Vercel does when this app is deployed there). I guess, why do we need any new commands then? Seems like we already have everything we need:

- For build: `build`
- For start: `next:start`

Keep in mind this starter is merely the output of our JSS initializer templates (i.e. [here](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/nextjs/package.json#L73)) and ideally we don't want to maintain 2 different flavors of the scripts.

The `start:*` commands are mainly intended for developers anyway, so this releases them back to those focuses.

Reply:
@ambrauer  can you please review?